packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/AddBankStatus/template.error.general.tsx

```diff
@@ -71,6 +80,21 @@ const BankLinkError: React.FC<Props> = ({
               id='copy.bank_linked_error_title_yourbank'
               defaultMessage='Is this your bank?'
             />
+          ) : bankStatus === 'BANK_TRANSFER_ACCOUNT_EXPIRED' ? (
+            <FormattedMessage
+              id='copy.bank_linked_error_title_expiredaccount'
+              defaultMessage='Expired Account Access'
+            />
+          ) : bankStatus === 'BANK_TRANSFER_ACCOUNT_REJECTED' ? (
+            <FormattedMessage
+              id='copy.bank_linked_error_title_connectionrejected'
+              defaultMessage='Connection Rejected'
+            />
+          ) : bankStatus === 'BANK_TRANSFER_ACCOUNT_FAILED' ? (
+            <FormattedMessage
+              id='copy.bank_linked_error_title_failedconnection'
+              defaultMessage='Failed Connection Request'
+            />
```

Comment:
maybe we can move all of bank statuses to be enum and then we can just reuse it all around 

```
export enum BANK_STATUSES {
  BANK_TRANSFER_ACCOUNT_EXPIRED = 'BANK_TRANSFER_ACCOUNT_EXPIRED',
  BANK_TRANSFER_ACCOUNT_REJECTED = 'BANK_TRANSFER_ACCOUNT_REJECTED',
 // etc...
}
```

and then just use 

```
bankStatus === BANK_STATUSES.BANK_TRANSFER_ACCOUNT_REJECTED
```

Reply:
Done, thanks!